assets/scss/utilities/_util-layout.scss

```diff
@@ -0,0 +1,4 @@
+// Spacing
+.u-margin-bottom {
```

Comment:
This could be better named. What if someone wants to use a smaller/bigger margin?